src/components/Header/index.js

```diff
@@ -112,4 +110,10 @@ class Header extends Component {
   }
 }
 
+Header.defaultProps = {
```

Comment:
I believe `.defaultProps` comes directly from [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values). So unless we're importing the library, I reckon this is basically the same as comments, no?

Reply:
Not exactly, defaultProps is built in React components, which are props used as fallback when the parent-provided props are undefined, see https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values. 

The Proptypes lib allows for typechecking, which would be useful for warning the dev if a required value is missing, amongst other things. 

Does this make sense to you?

Reply:
Ah yeah. That's the same file that I linked to. But I saw this and misunderstood:
> React.PropTypes has moved into a different package since React v15.5. Please use the prop-types library instead.

`.defaultProps` is still built into React? :+1:

Reply:
Yes ;)